build/pause/Makefile

```diff
@@ -16,8 +16,12 @@
 
 TAG = 2.0
 
-ARCH = amd64
-IMAGE = pause
+ARCH = $(shell go env GOARCH)
+ifeq ("${ARCH}", "amd64")
+	IMAGE = pause
+else
+	IMAGE = pause-${ARCH}
+endif
 # ARCH = arm
```

Comment:
@luxas which lines should I remove? Are we going to have pause-amd64 image? But don't we need that for backward compatibility?